calendar-bundle/src/Resources/contao/modules/ModuleEventReader.php

```diff
@@ -87,8 +87,18 @@ protected function compile()
 		global $objPage;
 
 		$this->Template->event = '';
-		$this->Template->referer = 'javascript:history.go(-1)';
-		$this->Template->back = $GLOBALS['TL_LANG']['MSC']['goBack'];
+
+		if ($this->overviewPage)
+		{
+			$this->Template->referer = PageModel::findById($this->overviewPage)->getFrontendUrl();
+			$this->Template->back = $GLOBALS['TL_LANG']['MSC']['eventOverview'];
+		}
+		else
+		{
+			// Backwards compatibility
```

Comment:
should we add a deprecation warning that this back link will be removed in Contao 5 if no overview link is set?

Reply:
Added in 198e15bed06b31dcfb8abc466c75fb1307a9c7d9.